README.md

```diff
@@ -57,15 +57,15 @@ We have a lot of resources to learn and understand how noname works:
 Once [noname is installed on your system](#installation), use `noname new` to create a project in a new directory, or `noname init` to initialize an existing directory. For example:
 
 ```
-$ noname new --path my_project
+noname new --path my_project
 ```
 
 This will create a `Noname.toml` manifest file, which contains the name of your project (which must follow a Github `user/repo` format) as well as dependencies you're using (following the same format, as they are retrieved from Github).
 
 This will also create a `src` directory, which contains a `main.no` file, which is the entry point of your program. If you want to create a library, pass the `--lib` flag to the `new` or `init` command of `noname`, and it will create a `lib.no` file instead.
 
 ```
-$ tree
```

Comment:
I think the `tree` indicates a command to list the folder content :D

Reply:
Adjusted now.